Geometry/CommonTopologies/src/GEMStripTopology.cc

```diff
@@ -0,0 +1,125 @@
+/**GEMStripTopology
+ * based on CSCRadialStripTopology and TrapezoidalStripTopology
+ * \author Hyunyong Kim - TAMU
+ */
+#include "Geometry/CommonTopologies/interface/GEMStripTopology.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include <iostream>
+#include <cmath>
+#include <algorithm>
+
+GEMStripTopology::GEMStripTopology(int ns, float aw, float dh, float r0)
+    : numberOfStrips_(ns), angularWidth_(aw), detHeight_(dh), centreToIntersection_(r0) {
```

Comment:
It would be good to add asserts to ensure valid values for these parameters. In particular, make sure `angularWidth_` is not zero since it is used later in division.
```
assert(angularWidth_ != 0);`

Reply:
`detHeight_` also should be checked for 0.

Reply:
yes, I added the assert function.